notice_and_comment/static/regulations/css/less/fonts.less

```diff
@@ -0,0 +1,29 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.less
+ */
+
+@font-face {
```

Comment:
Ah yeah the thing I meant was doing 

```css 
@font-face {
    font-family: 'Source Sans Pro';
    font-style: normal;
    font-weight: 200;
    src: url("../font/SourceSansPro-Light.ttf");
 }
 
@font-face {
    font-family: 'Source Sans Pro';
    font-style: normal;
    font-weight: 400;
    src: url("../font/SourceSansPro-Semibold.ttf");
 }

h1,
h2,
h3,
h4,
h5,
h6 {
    font-family: 'Source Sans Pro';
    font-weight: 400;
    margin-bottom: 0;
}

h1 {
    font-family: 'Source Sans Pro';
    font-weight: 200;
    font-size: 44px;
    line-height: 50px;
}
```

but it all comes out the same in the end right?